public/app/core/components/PageNew/PluginPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from 'react';
+
+import { PluginPageLayout, PluginPageProps } from '@grafana/runtime';
+import { PluginPageContext } from 'app/features/plugins/components/PluginPageContext';
+
+import { Page } from '../Page/Page';
+import { PageLayoutType } from '../Page/types';
+
+export function PluginPage({ children, pageNav, layout }: PluginPageProps) {
+  const context = useContext(PluginPageContext);
+  const pageLayout = getPageLayout(layout);
+
+  return (
+    <Page navModel={context.sectionNav} pageNav={pageNav} layout={pageLayout}>
+      <Page.Contents>{children}</Page.Contents>
+    </Page>
+  );
+}
+
+function getPageLayout(layout: PluginPageLayout | undefined): PageLayoutType {
+  if (layout && layout === PluginPageLayout.Canvas) {
+    return PageLayoutType.Canvas;
+  }
+
+  return PageLayoutType.Standard;
+}
```

Comment:
this feels a little weird cause they're basically the same type - could we move `PageLayoutType` into a place where both `Page` and `PluginPage` can access it (i guess `@grafana/data`?) and just use that type for both? 🤔

Reply:
yea it's a bit strange, was mainly to decouple things and make it easier to change things without impacting plugins. But think your right this should just be the same type